### PR TITLE
fixit: add type hints, descriptions and fix return vals for /translate

### PR DESCRIPTION
**Pull request by yil532**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: translate/samples/snippets/hybrid_glossaries/hybrid_tutorial_test.py [hunk 1]
```diff
@@ -78,28 +82,28 @@ def test_translate_glossary():
...
+    text = hybrid_tutorial.translate_text(input_text, "fr", "en", PROJECT_ID, created_glossary_name)
 
     assert text == expected_text
 
 
 # TEXT-TO-SPEECH TESTS
 
 
-def test_tts_standard(capsys):
+def test_tts_standard(capsys: pytest.LogCaptureFixture) -> None:
```
Comment (rsamborski) [inline translate/samples/snippets/hybrid_glossaries/hybrid_tutorial_test.py:95]: This causes a CI test failure:
```
hybrid_tutorial_test.py:95: in <module>
    def test_tts_standard(capsys: pytest.LogCaptureFixture) -> None:
E   NameError: name 'pytest' is not defined
```

**Review by rsamborski — APPROVED** [review]
Approved after fixing the CI/CD issues.

File: translate/samples/snippets/hybrid_glossaries/hybrid_tutorial_test.py (hunk 1 above)
Comment (yil532) [reply translate/samples/snippets/hybrid_glossaries/hybrid_tutorial_test.py:97]: Edited.

**Review by engelke — APPROVED** [review]
LGTM once lint issues are resolved.

**Comment by yil532** [comment]
Hi Charles, do you know which order of import I should put in? I kept changing back and forth of the `import pytest` and either of them works for me. https://screenshot.googleplex.com/38FqBH6u83kXsSc @engelke

**Comment by rsamborski** [comment]
> Hi Charles, do you know which order of import I should put in? I kept changing back and forth of the `import pytest` and either of them works for me. @engelke

I think the linter is looking for a following order of imports:
```
import stdlibs

import localpackages

import 3rdparty
```

In your case hopefully this code will work:
```
import os
import sys
import uuid

import hybrid_tutorial

import pytest
```

**Comment by yil532** [comment]
@dandhlee Nope... As we discussed last night, it still doesn't work

**Review by davidcavazos — APPROVED** [review]

**Pull request merged**